tests/test_quantum_gradients.py

```diff
@@ -281,6 +281,48 @@ def circuit(x, y):
         self.assertAllAlmostEqual(grad_A, grad_F, delta=self.tol)
         self.assertAllAlmostEqual(grad_A2, grad_F, delta=self.tol)
 
+    def test_CVOperation_with_heisenberg_and_no_params(self):
+        """An integration test for CV gates that support analytic differentiation
+        if succeeding the gate to be differentiated, but cannot be differentiated
+        themselves (for example, they may be Gaussian but accept no parameters).
+
+        This ensures that, assuming their _heisenberg_rep is defined, the quantum
+        gradient analytic method can still be used, and returns the correct result."""
+        self.logTestName()
+
+        for cls in qml.ops.cv.all_ops:
+            if cls.supports_heisenberg and (not cls.supports_analytic):
+                dev = qml.device('default.gaussian', wires=2)
+
+                U = np.array([[0.51310276+0.81702166j, 0.13649626+0.22487759j],
+                              [0.26300233+0.00556194j, -0.96414101-0.03508489j]])
+
+                if cls.num_wires == 0:
+                    w = list(range(2))
```

Comment:
seems silly to not just put `w=[0,1]`

Reply:
Also, there seems to be an implicit assumption here that any class that survives this far will support a 2x2 array as parameter (in fact, the notation `U` also suggests that this class can only be a unitary)

Reply:
This should get fixed for all tests as per #99